packages/web-config/webpack.config.base.ts

```diff
@@ -44,18 +44,18 @@ const optimization = {
 
 const stats = {
   errors: true,
-  warningsFilter: /node_modules\/mocha\/lib\/mocha.js/,
   warnings: true,
   all: false,
   builtAt: true,
   colors: true,
   modules: true,
-  maxModules: 20,
   excludeModules: /(main|test-entry).scss/,
   timings: true,
 }
 
-function makeSassLoaders ({ modules }: { modules: boolean }): RuleSetRule {
+const ignoreWarnings = [/node_modules\/mocha\/lib\/mocha.js/]
```

Comment:
Curious what we are ignoring? I remember mocha having some weird issues, but forgot what they were.

Reply:
I am not sure here to be honest. I just migrated the config option over to the v5 equivalent.

Reply:
Ah I found it. I don't think its working properly though so I will get this fixed.
<img width="441" alt="Screenshot 2023-08-02 at 7 48 10 AM" src="https://github.com/cypress-io/cypress/assets/3980464/21003baf-c9f9-41ce-99ce-bc3355f9f346">

Reply:
config should be 
```
const ignoreWarnings = [{
  module: /node_modules\/mocha\/lib\/mocha.js/,
}]
```

Reply:
fixed in 52312cf610